scss/_alert.scss

```diff
@@ -79,7 +79,7 @@
 //
 
 //// Additional content
-.alert * p {
+.alert p {
```

Comment:
As discussed, this may cause strange behaviours xith the following rule:
``
.alert-icon+* {
    margin: 0.125rem 0;
}
``